### PR TITLE
[IOTDB-697] add enablePartition in config

### PR DESCRIPTION
**Pull request by qiaojialin**
Disable partition by default, then the query performance will not decrease in 0.10. 
Also, the default concurrent write partition is set to 1.

### CODE REVIEW

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 1]
```diff
@@ -212,7 +216,7 @@ chunk_buffer_pool_enable=false
 # For example, your partitionInterval is 86400 and you want to insert data in 5 different days,
 # you should set this param >= 5
 # default number is 5
```
Comment (samperson1997) [inline server/src/assembly/resources/conf/iotdb-engine.properties:218]:
```suggestion
# default number is 1
```

Comment (qiaojialin) [reply server/src/assembly/resources/conf/iotdb-engine.properties:218]: fixed

**Comment by qiaojialin** [comment]
> The line 519 in `IoTDBConfig.java` is `private int concurrentWritingTimePartition = 10;`.
> Shouldn't we change the number to 1?

Yes, fixed

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java [hunk 2]
```diff
@@ -37,182 +40,242 @@
...
+      logger.error("Wrong " + TIMESTAMP_PRECISION_STRING + ", please set as: " + properties
+          .getProperty(TIMESTAMP_PRECISION_STRING) + " !");
       System.exit(-1);
     }
-    if (!(properties.getProperty("tsfile_storage_fs").equals(tsfileFileSystem))) {
-      logger.error("Wrong tsfile file system, please set as: " + properties
-              .getProperty("tsfile_storage_fs") + " !");
+
+    if (!(Long.parseLong(properties.getProperty(PARTITION_INTERVAL_STRING))
+        == partitionInterval)) {
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java:251]:
```suggestion
    if (Long.parseLong(properties.getProperty(PARTITION_INTERVAL_STRING))
        != partitionInterval) {
```

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java [hunk 3]
```diff
@@ -37,182 +40,242 @@
...
-  }
 
-  private void upgradeMlog() {
-    try {
-      MLogWriter.upgradeMLog(SCHEMA_DIR, MetadataConstant.METADATA_LOG);
-    } catch (IOException e) {
-      logger.error("Upgrade mlog.txt from {} failed.", SCHEMA_DIR, e);
+    if (!(properties.getProperty(TSFILE_FILE_SYSTEM_STRING).equals(tsfileFileSystem))) {
+      logger.error("Wrong " + TSFILE_FILE_SYSTEM_STRING + ", please set as: " + properties
+          .getProperty(TSFILE_FILE_SYSTEM_STRING) + " !");
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java:259]:
```suggestion
      logger.error("Wrong {}, please set as: {} !", TSFILE_FILE_SYSTEM_STRING, properties
          .getProperty(TSFILE_FILE_SYSTEM_STRING));
```

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java [hunk 4]
```diff
@@ -37,182 +40,242 @@
...
+      }
     }
-    if (!(Long.parseLong(properties.getProperty("storage_group_time_range"))
-            == partitionInterval)) {
-      logger.error("Wrong storage group time range, please set as: " + properties
-              .getProperty("storage_group_time_range") + " !");
+
+    if (!properties.getProperty(TIMESTAMP_PRECISION_STRING).equals(timestampPrecision)) {
+      logger.error("Wrong " + TIMESTAMP_PRECISION_STRING + ", please set as: " + properties
+          .getProperty(TIMESTAMP_PRECISION_STRING) + " !");
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java:246]:
```suggestion
      logger.error("Wrong {}, please set as: {} !", TIMESTAMP_PRECISION_STRING, properties
          .getProperty(TIMESTAMP_PRECISION_STRING));
```

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java (hunk 3 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java:259]: fixed

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java (hunk 2 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java]: fixed

File: server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java (hunk 4 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java]: fixed

**Pull request merged**